widget/form.go

```diff
@@ -119,6 +124,25 @@ func (f *Form) Disabled() bool {
 	return f.disabled
 }
 
+// SetOnValidationChanged is intended for parent widgets or containers to hook into the validation.
+func (f *Form) SetOnValidationChanged(callback func(error)) {
+	if callback != nil {
+		f.onValidationChanged = callback
+	}
+}
```

Comment:
In https://github.com/fyne-io/fyne/pull/2967/commits/23a0d89a27f50352396bc1005af0fa766e2a7595 I removed this from the entry. Don't you also think that it perhaps makes sense to allow removing the callback? This a comment not a request. I want to know your opinion :)

Reply:
It's a good question, and I don't see why not. But it also leads to a bigger question, should it be able to have multiple callbacks (perhaps this question (of multiple) should move to #2964)?

Reply:
I think multiple ones would have to occur at a later date. We can start small at first, without introducing breaking changes into the API.

Reply:
fair enough, I'll add that and the corresponding test later tonight.